config/puma.rb

```diff
@@ -10,7 +10,7 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port        ENV.fetch("PORT") { 3000 }
+port ENV.fetch("PORT") { 3000 }
```

Comment:
@bronzdoc `standard` complains in my local environment about lines like this one: 

```
standard: Use Ruby Standard Style (https://github.com/testdouble/standard)
standard: Run `rake standard:fix` to automatically fix some problems.
  config/puma.rb:7:25: Style/RedundantFetchBlock: Use `fetch("RAILS_MAX_THREADS", 5)` instead of `fetch("RAILS_MAX_THREADS") { 5 }`.
  config/puma.rb:13:10: Style/RedundantFetchBlock: Use `fetch("PORT", 3000)` instead of `fetch("PORT") { 3000 }`.
```